engine/games/card_flip.asm

```diff
@@ -1,5 +1,5 @@
-CARDFLIP_LIGHT_OFF EQU $ef
-CARDFLIP_LIGHT_ON  EQU $f5
+CARDFLIP_LIGHT_OFF EQUS "\"♂\"" ; $ef
+CARDFLIP_LIGHT_ON  EQUS "\"♀\"" ; $f5
```

Comment:
This seems like an assembler bug. `EQU "♂"` should be allowed.

Reply:
True; but until then the escapes are necessary, and were already used in some other files.